force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -228,19 +264,25 @@ public with sharing class ClassesObjectsMethods {
     public class Book {
         // Declare three public instance variables - title, author, pages
         // title
+        public String title;
         // author
-        // pages
-    
+        public String author;
+        // pages    
+        public Integer pages;
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(String title, String author) {
             // Initialize the title and author
+            this.title = title;
+            this.author = author;
         }
     
         // Method that returns the details of the book
         public String getBookDetails() {
             // return A string containing the details of the book in the format "Title: <title>, Author: <author>"
-            return null; // Replace null with the variable you used to store the result
+            String details = 'Title: '+ title + ', Author: ' + author;
```

Comment:
Be sure to use the this keyword to directly reference the instance value of the variables:
`String details = 'Title: '+ this.title + ', Author: ' + thisauthor;`